src/history_behaviour.rs

```diff
@@ -7,7 +7,7 @@ use crate::{COLOR_BOT};
 /**
  * Formats the watch list hash map as a movie history and sends it as an embedded message
  */
-pub fn show_history(bot_data: &mut crate::BotData, order: String) {
+pub fn show_history(bot_data: &mut crate::BotData, order: String, reverse: Bool) {
```

Comment:
```suggestion
pub fn show_history(bot_data: &mut crate::BotData, order: String, reverse: bool) {
```

The native type is written with a lowercase `b`